test/identifier/multiple-statement.spec.ts

```diff
@@ -127,6 +127,63 @@ describe('identifier', () => {
       expect(actual).to.eql(expected);
     });
 
+    describe('identifying statements with anonymous blocks', () => {
+      it('should identify a create table then a block', () => {
```

Comment:
why have this test repeated here and in `parser/oracle.spec.ts`?

Reply:
Just as an idiot check to be honest. Costs very little, gives me some assurances